theme/components/linkform.scss

```diff
@@ -24,12 +29,17 @@
 			.ck-button {
 				float: right;
 
+				// "Save" and "Cancel" buttons.
 				& + .ck-button {
 					margin-right: ck-spacing( 'medium' );
+				}
+
+				// The "Unlink" button.
+				&:last-child {
+					float: left;
```

Comment:
I think we should replace floats by a flex-box and remove overflow: hidden prom the parent element, but this is not the point of this PR.

Reply:
Flex-box is only partially supported in IE11, that's why we don't use it in the UI.